### PR TITLE
Made UI improvements #7

### LINKED ISSUE #7: Enhance UI
Right now the site is not pretty

Also one cool thing to do would be making the game bigger in the site (responsively of course)

### PR DESCRIPTION
**Pull request by anubhav1206**
Added a background color, styled the 'play' button and replaced the dropdown box with buttons with identical styles as the 'play' button.
image

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -26,15 +26,78 @@ span {
   text-transform: capitalize;
   text-shadow: 0 0 1px #fff;
 }
+.video-game-button {
+
```
Comment (datsfilipe) [inline src/App.css]: I think we can remove this line.
```suggestion
```

File: src/index.css [hunk 2]
```diff
@@ -37,25 +37,74 @@ h1 {
...
-button {
-  border-radius: 8px;
-  border: 1px solid transparent;
-  padding: 0.6em 1.2em;
-  font-size: 1em;
-  font-weight: 500;
-  font-family: inherit;
-  background-color: #1a1a1a;
+.play-button {
+
```
Comment (datsfilipe) [inline src/index.css]: This one also.
```suggestion
```

**Review by datsfilipe — CHANGES_REQUESTED** [review]
The styles of `.inputContainer button` in `src/components/BottomContent/BottomContent.module.css` file is conflicting with the `.video-game-button` radius. **Removing the line that applies the radius in `.inputContainer button` would fix that**.

After changing that, just accept the changes below.

**Comment by datsfilipe** [comment]
@anubhav1206 you did a very good job! Resolving the requested changes I will surely approve.

**Comment by anubhav1206** [comment]
> @anubhav1206 you did a very good job! Resolving the requested changes I will surely approve.

Thank You! I have made the requested changes.

**Review by datsfilipe — APPROVED** [review]
LGTM! thank you for the contribution.

**Pull request merged**